modules/serverless-admin-broker-default.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuring-eventing-defaults.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-broker-default_{context}"]
+= Configuring the default broker implementation
+
+You can specify the default broker implementation for the cluster, or for one or more namespaces.
+
+To do this, you must configure the `defaultBrokerClass` spec in the `KnativeEventing` custom resource (CR), as well as adding configuration for the `config-br-defaults` config map.
+
+Currently, the available broker class options for {ServerlessProductName} are:
+
+* The `MTChannelBasedBroker` class, which uses the default Knative multi-tenant, channel-based broker.
+
+* The `Kafka` broker class, which uses the Knative Kafka broker (currently Technology Preview).
+
+If no broker class is specified, the cluster uses the `MTChannelBasedBroker` class by default.
+
+.Prerequisites
+
+ifdef::openshift-enterprise[]
+* You have cluster administrator permissions on {product-title}.
+endif::[]
+
+ifdef::openshift-dedicated[]
+* You have cluster or dedicated administrator permissions on {product-title}.
+endif::[]
+
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+
+* Optional: If you want to use the Kafka broker, you must install Knative Kafka on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` CR to add the `defaultBrokerClass` spec and `config-br-defaults` config map details:
++
+.Example `KnativeEventing` CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  defaultBrokerClass: <default-broker-class> <1>
+  config:
+    config-br-defaults: <2>
+      default-br-config: |
+        clusterDefault: <3>
+          brokerClass: MTChannelBasedBroker
```

Comment:
this line is redundant and unfortunately ignored.

see: https://github.com/knative/operator/issues/813

Reply:
Should I remove `brokerClass: MTChannelBasedBroker` then?

Reply:
for both this and the namespaceDefaults or just for the clusterDefault?

Reply:
the `brokerClass` in general - b/c it is controlled by the `KnativeEventing` spec - and not the configmap

Reply:
Ahhhh very good! I was wondering if there was a difference re Operator vs ConfigMap, thank you